### DISCUSSION TITLE
Why doesn't the amount/concentration of CYP3A4 in the Liver Intracellular compartment change throughout my simulation?

### DISCUSSION THREAD

**Discussion by DonkeySaddle** [Open-Systems-Pharmacology/Forum#1603] [Questions & Problems] [answered]
Plots are flat (though nonzero) and unmoving? I understand the amount of 
CYP3A4  could be so large, that the drug doesn't have an impact, but given that there is an interaction by way of competitive inhibition with CYP3A4, I assumed the amount of CYP3A4 would change.

**Comment by RobinM92** [comment, marked as answer]
Competitive inhibition should not alter enzyme concentration (Km changes due to larger amounts of substrate being needed to reach the same reaction velocity when another compound is competing for binding to the reaction site). 

In PKSim, only irreversible (time-dependent) inhibition and induction impact the actual enzyme concentrations (by respectively increasing degradation and prouction), which you would see in the CYP-concentration-time plots. You can easily try this out yourself by implementing one of these interactions.

The underlying principles are explained nicely in the manual and supporting references: https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-defining-inhibition-induction-processes 

Good luck!

**Comment by DonkeySaddle** [comment]
This is awesome. Thanks so much!
